mscore/shortcut.cpp

```diff
@@ -2591,6 +2591,13 @@ Shortcut Shortcut::_sc[] = {
          QT_TRANSLATE_NOOP("action","Zoom to 100%"),
          QT_TRANSLATE_NOOP("action","Zoom to 100%")
          },
+      {
+         MsWidget::MAIN_WINDOW,
+         STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT | STATE_PLAY,
+         "zoom-page-width",
+         QT_TRANSLATE_NOOP("action","Zoom to page-width / previous magnification level"),
```

Comment:
Oh, it looks like this needs to be fixed also: maybe merely "Zoom: Page-Width" would work to keep it shorter as with most others and give capitalization.